pkg/click/click

```diff
@@ -149,6 +160,10 @@ if [[ $EXECUTE -ne 0 ]]; then
         STATUS=1
     fi
 
+    if [[ $CARD -ne 0 ]]; then
+        echo "-c meaningless with -e; ignoring" >&2
+    fi
```

Comment:
`k`, and `p` options also make no sense when `c` is enabled

Reply:
added checks for both `k` and `p`